deepchecks/base/check.py

```diff
@@ -148,9 +148,10 @@ def _ipython_display_(self):
             display_html(f'<h4>{self.header}</h4>', raw=True)
         if self.check and '__doc__' in dir(self.check):
             docs = self.check.__doc__
-            # Take first non-whitespace line.
-            summary = next((s for s in docs.split('\n') if not re.match('^\\s*$', s)), '')
-            display_html(f'<p>{summary}</p>', raw=True)
+            if docs:
```

Comment:
a. why?
b. move the condition to the upper if instead of additional if

Reply:
a. the original didn't worked. the __docs__ was None
b. wanted to not make duplicate code. doesn't really matter

Reply:
I think that's matter, it's uglier to have `if` inside `if`
and it's not saving any work, since `self.check.__doc__` access a variable not a function call